scipy/sparse/linalg/_expm_multiply.py

```diff
@@ -350,49 +353,6 @@ def _expm_multiply_simple_core(A, B, t, mu, m_star, s, tol=None, balance=False):
         }
 
 
-def _onenormest_matrix_power(A, p,
```

Comment:
Having all this for a private one-liner that is only used once is not necessary IMO